python/py-buildbot-worker/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-buildbot-worker
+version             2.3.1
+revision            0
+
+maintainers         nomaintainer
+description         Buildbot Worker Daemon
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://buildbot.net/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             
```

Comment:
Missing license.

Reply:
Again, this is a question for @Korusuke.

Even if some strange licence is used (pypi says something about GPL which should not be too exotic), at least `unknown` should be returned here.

Reply:
buildbot uses GPL-2. I forgot to add it here

Reply:
Sorry for this, I will update the port soon in which the license name conversion feature is now available.